src/Service.Tests/SqlTests/RestApiTests/Put/PutApiTestBase.cs

```diff
@@ -829,6 +829,57 @@ public virtual async Task PutOneWithNonNullableFieldAsNull()
             );
         }
 
+        /// <summary>
+        /// Verifies that we throw an exception when an extraneous field that does not map to a backing column in the table
+        /// is provided in the request body for a PUT operation. This test validates the behavior of rest.request-body-strict when it is:
+        /// 1. Included in runtime config (and set to true)
+        /// 2. Excluded from runtime config(defaults to true)
+        /// </summary>
+        [TestMethod]
+        public async Task PutOneTestWithExtraneousFieldsInRequestBody()
```

Comment:
can you note the significance of testing against entity with non-autogenerated versus autogenerated PK?

Reply:
There is no difference, can remove one.